app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue

```diff
@@ -171,6 +180,138 @@ export default {
         },
       ];
     },
+    portals() {
```

Comment:
Maybe this sample portal data can be added in the stories file

Reply:
@fayazara Done https://github.com/chatwoot/chatwoot/pull/5126/commits/0231ff8baec0b0dbca9e880d80a0400427bf7fc0